get_channel_ids.py

```diff
@@ -1,35 +1,47 @@
-from telethon import TelegramClient
-
-import logging
-logging.basicConfig(format='[%(levelname) 5s/%(asctime)s] %(name)s: %(message)s',
-                    level=logging.WARNING)
+from telethon import TelegramClient, events
+import json
+from shutil import copyfile
+import os
+from dotenv import load_dotenv
 
+load_dotenv()
 
 # Use your own values from my.telegram.org
-api_id = -1
-api_hash = ''
-client = TelegramClient('anon', api_id, api_hash)
-
-async def main():
-    # Getting information about yourself
-    me = await client.get_me()
-
-    # "me" is a user object. You can pretty-print
-    # any Telegram object with the "stringify" method:
-    print(me.stringify())
-
-    # When you print something, you see a representation of it.
-    # You can access all attributes of Telegram objects with
-    # the dot operator. For example, to get the username:
-    username = me.username
-    print(username)
-    print(me.phone)
-
-    # You can print all the dialogs/conversations that you are part of:
-    async for dialog in client.iter_dialogs():
-        print(dialog.name, 'has ID', dialog.id)
-
-    # await client.send_message('me', 'Hello, myself!')
-
-with client:
-    client.loop.run_until_complete(main())
+api_id = os.getenv("API_ID")
+api_hash = os.getenv("API_HASH")
+bot_token = os.getenv("BOT_TOKEN")
+# bot_name = os.getenv("BOT_NAME")
+client = TelegramClient("anon", api_id, api_hash)
+
+
+@client.on(events.ChatAction)
+async def chat_action(event):
```

Comment:
I guess it's not strictly necessary, but we might want to filter these events to only include those where an admin adds the bot, rather than adds anyone else.